Resources/Markdown/try-package.md

```diff
@@ -0,0 +1,17 @@
+---
+page-title: Try in a Playground
+description: Try any Swift package in a Swift Playground by downloading the Swift Package Index Playgrounds app for macOS.
+---
+
+[download-url]: https://github.com/SwiftPackageIndex/SPI-Playgrounds-Releases/raw/main/releases/SPI-Playgrounds-0.2.0.app.zip
```

Comment:
This feels like it could be frustrating to update in the future - could we duplicate 0.2.0 and call it `SPI-Playgrounds-latest.app.zip` and update that file every time in conjunction with storing each version?

Reply:
Yea we've been chatting about this today and it won't ship like this. 👍

Reply:
I haven't found a good way to create a redirect or symbolic link to the latest version so I've simply updated our release `Makefile` to take care of the packaging.

It's now as simple as placing the `.app` bundle at the top level, running `make release VERSION=x.y.z`, and then doing `git push`.

The `make` command will copy the zip to `./releases/SPI-Playgrounds-$VERSION.app.zip` for the appcast feed but leave `SPI-Playgrounds.app.zip` at the top level for the latest release download.

The URL for is then: https://github.com/SwiftPackageIndex/SPI-Playgrounds-Releases/raw/main/SPI-Playgrounds.app.zip

I've just pushed 0.3.0 where all this is in place, so this URL is live.

Reply:
One thing we should check: is it ok to use GH as a file share / update service? I wouldn't think we're going to create a lot of traffic but it might trip something up.

Reply:
> One thing we should check: is it ok to use GH as a file share / update service? I wouldn't think we're going to create a lot of traffic but it might trip something up.

I think we'll be fine, and if we get a warning (which I don't think we will) we can move it to the Azure CDN.